### DISCUSSION TITLE
`shinystan`-like interface for interacting with draws

### DISCUSSION THREAD

**Discussion by torfjelde** [TuringLang/Turing.jl#2397] [Ideas]
shinystan is a web-frontend for interacting with the results of draws in stan in an intuitive way.

The "closets" thing we have to this in Turing.jl is TuringCallbacks.jl with tensorboard, but this is still pretty far away from shinystan.

So:
1. Should we have something similar?
2. Should we document more clearly the current TuringCallbacks.jl?

**Comment by SamuelBrand1** [comment]
Yes please on current `TuringCallbacks`!
